### DISCUSSION TITLE
Openstick

### DISCUSSION THREAD

**Discussion by fab1an2** [markqvist/Reticulum#219] [Help & Questions]
https://extrowerk.com/2022/07/31/OpenStick/

This device are cheap, small and have normal linux with python. Linux have many bugs and not working ideal (i canot setup WPA3), but this is very interesting.

Is posible connect esp lora device trought wifi to other device with wifi like this? I think about adhoc wifi mesh network. (no wifi 6)

**Comment by srem1** [comment]
I have one. This dev no have lora. Only wifi
